### PR TITLE
No fun allowed

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

Wizards can no longer use Summon Events, Summon Magic, or Summon Guns

#### Changelog

:cl:  
rscdel: rip summons
/:cl:

### CODE REVIEW

**Comment by ThatLing** [comment]
Where's the reason for this PR?

**Comment by ThatLing** [comment]
Also, why straight up removing it instead of just disabling it in the config?

**Comment by Amelia0010** [comment]
@ThatLing 

Config: 
I'm moving a bunch of things which really are gameplay/headcoder decisions over to the github rather than the config, the only reason they're in it to begin with is TG's inability to co-operate Headmins and Headcoders between, and the fact that they had to serve multiple communities and servers. We have one server, and one community, and so stuff like "should X item be enabled" should be a code-issue, not an admin issue"

As for the intent:
The summons are a massive pain in the arse for everyone involved, Summon Guns is admin hell, because Bobby McGreyshit feels entitled to use the insta-death rifle he was just given, Summon Magic is pretty much the same thing, and Summon Magic Events is like a bad admin's bus; "ayy lmao what if we just burn everyone unless they get on a table"

**Comment by Amelia0010** [comment]
Also I made a removal PRs milestone so we have a full list of PRs to remove stuff, from this PR on.

File: code/modules/mob/dead/new_player/new_player.dm [hunk 1]
```diff
@@ -390,13 +390,6 @@
 			to_chat(humanc, "<span class='userdanger'><i>THERE CAN BE ONLY ONE!!!</i></span>")
 			humanc.make_scottish()
 
-		if(GLOB.summon_guns_triggered)
-			give_guns(humanc)
-		if(GLOB.summon_magic_triggered)
-			give_magic(humanc)
-		if(GLOB.curse_of_madness_triggered)
```
Comment (SomeguyManperson) [inline code/modules/mob/dead/new_player/new_player.dm:397]: If we aren't deleting curse of madness this probably needs to stay

**Comment by boodaliboo** [comment]
staff in staff public said it causes the same ammount of tickets as gang or cult already does which isnt that bad

**Comment by Sir-Lagsalot** [comment]
It's not about tickets, it's because a lot of people have been complaining about event wizards being really unfun, which I frankly agree with.

**Comment by LoliconSlayer** [comment]
Summon guns: Do you really think the _entire_ crew with guns isn't gonna turn into a shitfest?

Summon magic: Sure there's a potential downside in trying to read the magic books but it's pretty ass when there's some POS using his spells to do things like create doors everywhere just because they can.

Summon events: Boy I sure do love joining as an engineer only to be swapped with 3 different bodies, forced to change race, and forced to wear the cursed luchador mask only for the station to get vented so I can't wear internals.

The amount of tickets doesn't matter because most of the bullshit that goes down during summon events is valid, it's about the fact that a single wizard can just put all their points into summon events and ruin an entire round for probably like 80%-95% of the players.

**Comment by Oakboscage** [comment]
Mane this is like the best part of wizards

**Comment by Darkstick** [comment]
Good thing. Event wizard is uncreative and dumb, yet 80% of the wizards are going to pick only the event spells. All they're good for is making boring chaotic extended rounds, and those get really old after literally every fucking wizard picks them.

Forcing the community out of the "all events" comfort zone would bring back wizard rounds to it's former juggernaut-esque glory rather than this no-rp snore fest of being given a bunch of guns that you're not allowed to use and getting randomly race changed for no reason.

**Comment by Sir-Lagsalot** [comment]
Well, Oak DID play two rounds a few days ago, so I think he's the most qualified to comment on this. Shut the pr down guys, Oak thinks it's fun.

**Comment by Amelia0010** [comment]
Oak thinks it's fun, but no fun allowed. Sounds like you just made a great case for removing this.

**Comment by ynot01** [comment]
this is so sad can we merge this

**Comment by Sir-Lagsalot** [comment]
Defaultdance.gif

**Comment by Smudged** [comment]
This is the good ending.

**Pull request merged**